client/my-sites/site-settings/section-import.jsx

```diff
@@ -142,30 +153,37 @@ class SiteSettingsImport extends Component {
 	}
 
 	/**
-	 * Receives import jobs data (`importsForSite`) and maps this to return a
-	 * list of importer elements for active import jobs
+	 * Receives import jobs data (`activeImporter`) and returns an
+	 * importer view for that importer
 	 *
-	 * @param {Array} importsForSite The list of active import jobs
-	 * @returns {Array} Importer react elements for the active import jobs
+	 * @param {Object} activeImporter
+	 * @returns {Component||null} Importer Component for the active import job
 	 */
-	renderActiveImporters( importsForSite ) {
-		return importers.map( importer => {
-			const { type, isImporterEnabled, component: ImporterComponent } = importer;
+	renderImporterScreen( activeImporter ) {
+		const { importerOption, site } = this.props;
+		const selectedImportType = activeImporter.type || importerOption;
```

Comment:
We should guard against reference errors when `activeImporter` is null or undefined.

I tried to do:

`dispatch( { type: "IMPORTS_IMPORTER_OPTION_SELECT", importerOption: 'gibberish' } )` & this crashed.